spec/macros/duke/urls_spec.rb

```diff
@@ -54,5 +54,18 @@
         )
       end
     end
+
+    context '944 fields:' do
+      it 'correctly detects a MARC 944 field and sets the URL correctly' do
+        rec = make_rec
+        rec << MARC::DataField.new('944', '0', ' ',
+                                  ['b', '61903212100008501']
+                                  )
+        result = run_traject_on_record('duke', rec)
+        expect(JSON.parse(result['url'][0])['href']).to(
+          include('61903212100008501')
```

Comment:
I'd recommend matching on the full URL here since it'd demonstrate more clearly what kind of URL these fields will yield. I.e.,
```
expect(JSON.parse(result['url'][0])['href']).to(
          eq('https://duke.userservices.exlibrisgroup.com/discovery/openurl?institution=01DUKE_INST&vid=01DUKE_INST:Services&rfr_id=info:sid%2Fsummon&rft_dat=ie%3D61903212100008501')
)
```